policy/modules/system/systemd.if

```diff
@@ -274,6 +275,25 @@ interface(`systemd_user_unix_stream_activated_socket',`
 	systemd_user_activated_sock_file($2)
 ')
 
+#######################################
+## <summary>
+##  Allow the specified domain to write to
+##  systemd-notify socket
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`systemd_write_notify_socket',`
+	gen_require(`
+		type systemd_runtime_notify_t;
+	')
+
+	allow $1 systemd_runtime_notify_t:sock_file write;
```

Comment:
It needs:
* the dir accesses to get to the sock_file
* a unix socket connectto/sendto init_t.

Reply:
Ok, I did both of these.  The dir access makes sense.  I'm not sure I understand the unix socket change and I think I did what you are asking.  Of course, if I misunderstood, let me know and I will alter it.

Reply:
Ok, just the sendto.  Hopefully this is now good.